src/Layout/Tutorial/Tutorial.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+export interface TutorialProps {
+    showTutorial: boolean
+    closeTutorial: () => void
+}
+
+export class Tutorial extends React.Component<TutorialProps, {}> {
+
+    public render() {
+        const showTutorial = this.props.showTutorial ? " is-active" : "" ;
+        const modalClass = "modal" + showTutorial;
+
+        return (
+            <div className={modalClass}>
+            <div className="modal-background" onClick={() => this.props.closeTutorial()}></div>
+            <div className="modal-card">
+                <header className="modal-card-head">
+                <p className="modal-card-title">Velkommen til Lavinia</p>
+                <button className="delete" aria-label="close" onClick={() => this.props.closeTutorial()}></button>
```

Comment:
The button is quite small and on the wrong side of the screen for most users. I suggest centering it and making it wider so that it is more accessible and looks more symmetrical.